### PR TITLE
First part of Transfer processing

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This implements the first few steps of the Transfer Processing workflow:
- Creates a page showing all transfer records, with links to each transfer detail page
- Implements a jQuery plugin (DataTables) to allow interactive filtering and sorting of the list of Transfers

#### Developer

- [x] All new ENV is documented in README
- [x] All new ENV has been added to Heroku Pipeline, Staging and Prod
- [x] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)
- [ ] Stakeholder approval has been confirmed (or is not needed)

#### Code Reviewer

- [x] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [x] There are appropriate tests covering any new functionality
- [x] The documentation has been updated or is unnecessary
- [x] The changes have been verified
- [x] New dependencies are appropriate or there were no changes

#### Requires database migrations?

NO

#### Includes new or updated dependencies?

NO

### CODE REVIEW

File: app/views/transfer/select.html.erb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<%= content_for(:title, "Transfer Processing | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js"></script>
+<% end %>
+
+<link href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.min.css" rel="stylesheet">
```
Comment (JPrevost) [inline app/views/transfer/select.html.erb:7]: I wonder if our theme gem should allow `<% content_for :additional_css do %>` to better handle this use case?

(No action needed, just thinking out loud).

File: app/views/transfer/_empty.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<tr class="empty">
+  <td colspan="6">There were no transfers matching your parameters.
```
Comment (JPrevost) [inline app/views/transfer/_empty.html.erb:2]: `td` not closed

**Review by JPrevost — APPROVED** [review]

File: app/views/transfer/_empty.html.erb (hunk 2 above)
Comment (matt-bernhardt) [reply app/views/transfer/_empty.html.erb:2]: *smacks head* D'oh! I'm going to push something to fix this.

File: app/views/transfer/select.html.erb (hunk 1 above)
Comment (matt-bernhardt) [reply app/views/transfer/select.html.erb:7]: I was on the point of asking why I couldn't find a layout in this app - didn't think to ask about the theme gem. Yes, I think that would be a good feature to add for a use case like this.

(I don't think that blocks this merging, though right? We can do that via maintenance?)

**Review by JPrevost — APPROVED** [review]
Yeah that theme gem thing wasn't intended to be included in this work. :shipit:

**Pull request merged**